articles/blockchain-consensus-protocols/index.md

```diff
@@ -0,0 +1,83 @@
+# An Overview of Consensus Protocols in Blockchain
+
+An exciting application called Bitcoin was introduced after the great recession in 2008. This was one of the first real applications of blockchain technology. Also known as distributed ledger technology, blockchain stands out as a promising solution due to the following properties:
+
+- Security
+- Accountability
+- Distributed
+- Transparent 
+- Immutable
+
+This article will discuss the role of consensus protocols in blockchain technology and dive deeper into the various consensus protocols. 
+
+### Introduction
+
+Blockchain technology is a distributed ledger to record any transactions that happen over the network. The concept of decentralization is deeply engraved into the design of blockchain technology. Decentralization means there is no major entity keeping track of the transactions, and instead, it is distributed across many nodes in the network. The technology allows one to verify the transactions via cryptographic hash functions. The underlying assumption is that the record present with most nodes is likely to be the least falsible. 
```

Comment:
> it is distributed across many nodes in the network. The technology allows one to verify the transactions via cryptographic hash functions.

You could elaborate on Cryptographic Hash Function in a line or two, since a beginner might not be aware of it